### PR TITLE
Completed html-relative-links.

### PR DESCRIPTION
**Pull request by Mitchelllund**
html-relative-links

### CODE REVIEW

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="C:\Users\Mitchell\repos\m1-0122-code-solutions\html-relative-links\index.html">Home Page</a></li>
```
Comment (Zhiwutian) [inline html-relative-links/about-me/about-me.html:13]: These are absolute paths. So make sure to update them all to relative paths as per the reading. Make sure to reach out with questions if you have them!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Mitchell, make sure to update all of the paths to relative paths instead of Absolute paths. Reach out if you have questions!

**Comment by Mitchelllund** [comment]
htmlrelativelinks

**Comment by Mitchelllund** [comment]
Here is my updated html-relative-links. Thank you so much Brett for the help. I was overthinking that way to much, I made that way to difficult.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**